src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,97 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+
+    private int threshold;
+    private int size;
+    private Node<K,V>[] table;
+
+    public MyHashMap() {
+        threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+        size = 0;
+        table = (Node<K,V>[]) new Node[DEFAULT_CAPACITY];
+    }
+
+    private static class Node<K,V> {
+        final int hash;
+        final K key;
+        V value;
+        Node<K,V> next;
+
+        public Node(int hash, K key, V value, Node<K, V> next) {
+            this.hash = hash;
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
 
     @Override
     public void put(K key, V value) {
+        int index = getIndex(key);
+        Node<K,V> currentNode = table[index];
+        Node<K,V> nextNode = new Node<>(index, key, value, null);
+        if (currentNode == null) {
+            table[index] = nextNode;
+        } else {
+            while (true) {
+                if (key == currentNode.key || (key != null && key.equals(currentNode.key))) {
+                    currentNode.value = value;
+                    return;
+                } else if (currentNode.next == null) {
+                    currentNode.next = nextNode;
+                    break;
+                }
+                currentNode = currentNode.next;
 
+            }
+        }
+        size++;
```

Comment:
u don't need this increase, if u modify line 52

Reply:
done